pkg/cloudprovider/aws/cloudprovider.go

```diff
@@ -108,7 +108,7 @@ func withUserAgent(sess *session.Session) *session.Session {
 }
 
 // Create a node given the constraints.
-func (c *CloudProvider) Create(ctx context.Context, constraints *v1alpha5.Constraints, instanceTypes []cloudprovider.InstanceType, quantity int, callback func(*v1.Node) error) chan error {
+func (c *CloudProvider) Create(ctx context.Context, constraints *v1alpha5.Constraints, instanceTypes []cloudprovider.InstanceType, quantity int, callback func(*v1.Node) error) <-chan error {
```

Comment:
The original purpose of making this a channel (as opposed to a callback) was to allow cloud providers to implement arbitrary batching behavior. Instead, we implemented batching in vendor neutral binpacking code. I think we should sync on @bwagner5 on what the desired implementation strategy is.

I don't think this should necessarily block, but I wanted to bring it up since we're touching it.

Reply:
I think we actually want to keep this channel and refactor the way it's used currently. Right now, we receive from it right after calling which blocks per packing. Practically, it's not much overhead since the packing is batched now and we do have a concurrent work queue on schedules. But we could receive from the channels after we've called all the creates and actually utilize the concurrency this provides. 

TLDR: I think this is good :)